script_runner.py

```diff
@@ -78,6 +79,9 @@ def _run_command(self, cmd, working_dir):
 
         command_thread.start()
 
+        console_flush_thread = threading.Thread(target=self.__trigger_flush_console_output)
```

Comment:
class threading.Timer(interval, function, args=None, kwargs=None)
is the signature of the Timer object.